### PR TITLE
Release foreman-tasks 0.7.14 (RPM)

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by domcleal** [comment]
[test] as the wrong test results are associated.

I think the %files list is going to need a big update as there are new dot-files and directories.

**Comment by stbenjam** [comment]
Green now, thanks

File: rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 1]
```diff
@@ -159,6 +164,7 @@ exit 0
 %{foreman_apipie_cache_foreman}
 %{foreman_apipie_cache_plugin}
 %doc %{gem_instdir}/LICENSE
+%doc %{gem_instdir}/README.md
```
Comment (domcleal) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: This is already in the %doc subpackage, it shouldn't be in the main package.

Comment (stbenjam) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Sorry, I always forget README doesn't go in the main package

**Comment by domcleal** [comment]
Thanks @stbenjam, building to nightly only.

**Pull request merged**